packages/style-engine/src/styles/background/index.ts

```diff
@@ -16,6 +16,14 @@ const backgroundImage = {
 			return styleRules;
 		}
 
+		if ( typeof _backgroundImage === 'string' ) {
```

Comment:
TODO:
- add tests
- confirm that this is the right place to imply "file" for `backgroundImage.url` values
- test the `theme` source